src/components/Pages/Admin/View.vue

```diff
@@ -184,6 +185,10 @@ export default {
     },
 
     methods: {
+        groupColor(color) {
+            return(`background:rgb(${color[0]}, ${color[1]}, ${color[2]})`)
```

Comment:
I think a cleaner way to do this would be to use operator `...` on `color`. Also, please add space after the colon.